### DISCUSSION TITLE
MinGW-w64 still needed?

### DISCUSSION THREAD

**Discussion by theking2** [git-for-windows/git#3995] [General]
Is mingw-w64 still needed when not installing MiniTTY? the latest Terminal is quite adequate. It adds 0.5GByte

**Comment by rimrul** [comment]
I'm not sure I understand the question. We don't include `mingw-w64` in Git for Windows. 

Are you talking about the `/mingw64` folder?  That would fit the size description according to windows explorer. That folder is absolutely necessary, but about 460mb of the size reported should be miscalculated hardlinks on most systems.

**Reply by dscho** [reply]
From Git for Windows' [Known Issues](https://github.com/git-for-windows/build-extra/blob/main/ReleaseNotes.md#known-issues):

> Older versions of the Windows Explorer do not calculate Git for Windows' on-disk size correctly, as it is unaware of hard links. Therefore, it might look like Git for Windows takes up 1.5GB when in reality it is about a third of that.
